templates/scenarios/sso-tab-js/.vscode/tasks.json

```diff
@@ -2,45 +2,72 @@
     "version": "2.0.0",
     "tasks": [
         {
-            "label": "Pre Debug Check & Start All",
+            "label": "Start Teams App Locally",
             "dependsOn": [
-                "validate local prerequisites",
-                "prepare local environment",
-                "Start All"
+                "Validate prerequisites",
+                "Create resources",
+                "Set up local projects",
+                "Start services"
             ],
             "dependsOrder": "sequence"
         },
         {
-            "label": "validate local prerequisites",
-            "type": "shell",
-            "command": "exit ${command:fx-extension.validate-local-prerequisites}",
-            "presentation": {
-                "reveal": "never"
+            "label": "Validate prerequisites",
+            "type": "teamsfx",
+            "command": "debug-check-prerequisites",
+            "args": {
+                "prerequisites": [
+                    "nodejs", // Validate if Node.js is installed.
+                    "m365Account", // Sign-in prompt for Microsoft 365 account, then validate if the account enables the sideloading permission.
+                    "portOccupancy" // Validate available ports to ensure those debug ones are not occupied.
+                ],
+                "portOccupancy": [
+                    53000 // tab service port
+                ]
             }
         },
         {
-            "label": "prepare local environment",
-            "type": "shell",
-            "command": "exit ${command:fx-extension.pre-debug-check}",
-            "presentation": {
-                "reveal": "never"
+            "label": "Create resources",
+            "type": "teamsfx",
+            "command": "provision",
+            "args": {
+                "configFile": "${workspaceFolder}/teamsfx/app.local.yml",
```

Comment:
-> `template`

Reply:
Will updated in PR: https://github.com/OfficeDev/TeamsFx/pull/6578